java/clients/src/test/java/sleeper/clients/admin/testutils/AdminClientMockStoreBase.java

```diff
@@ -49,4 +49,14 @@ protected void setInstanceTables(InstanceProperties instanceProperties, String..
         setInstanceProperties(instanceProperties);
         when(store.listTables(instanceProperties.get(ID))).thenReturn(Arrays.asList(tableNames));
     }
+
+    protected void setStateStoreForTable(String tableName) {
```

Comment:
If we pass in the state store here, we can set it explicitly in tests that care about it (eg. number of partitions, leaves).